### ISSUE TITLE
New App Submission - CoinCircle Wallet

### ISSUE THREAD

**Issue by coopermaruyama** [WalletConnect/walletconnect-registry#379]
## App Logo (Required)

![CC_App_Icon-1024](https://user-images.githubusercontent.com/1325802/142136896-f88bbfc6-b2d7-4979-b307-b49f23b0f29c.png)

![CC_App_Icon-60@3x](https://user-images.githubusercontent.com/1325802/142136424-7164b8a5-b1c1-4ef9-a570-0869a403bd87.png)

## App Details (Required)

Type: Wallet
Name: CoinCircle
Description: Earn, Pay, Buy, Borrow Crypto
Homepage Url: https://coincircle.com

Chains supported: Ethereum Mainnet
Versions supported: 1.0

## App Url (Required - at least one)

Browser Url: https://coincircle.com
iOS Url: https://apps.apple.com/us/app/coincircle/id1470350344
Android Url: https://play.google.com/store/apps/details?id=com.coincircle
Mac Url:
Windows Url:
Linux Url:

## Mobile Linking (Optional)

Deep Link (iOS): com.coincircle.app://walletconnect
Deep Link (Android): com.coincircle://walletconnect
Universal Link: https://coincircle.com/app
Universal Link for WalletConnect: https://coincircle.com/app/walletconnect

## Desktop Linking (Optional)

Deep Link:
Universal Link:

## Metadata (Optional)

Short Name:
Primary Color: #2499c4
Secondary Color: #1c2d5a

**Comment by erickmiller** [comment]
Hello WalletConnect team, We'd love to have this merged when you guys get a chance?  Thanks!

**Comment by coopermaruyama** [comment]
@pedrouid bump on this sorry to bother.

**Comment by erickmiller** [comment]
hey @pedrouid am sure you're super busy, anything we can do to try to get our submission reviewed?  We added WalletConnect to our code & our app, it's pretty stable and working pretty well, our app has had WalletConnect functionality live and working in the app store for about a month now -- we've tested it with a ton of dapps and have handled all the edge cases I think pretty well -- any idea when you'll be able to get to this (or) maybe @grachyov could help?  Let us know if there's anything we can do.  Would love to see this happen, thanks in advance!   WalletConnect rocks.  Thanks
